### DISCUSSION TITLE
Fibaro FGD212 Dimmer 2 and FGS223 (double switch) stopped working

### DISCUSSION THREAD

**Discussion by bckmrk** [zwave-js/zwave-js#4976] [Q&A]
A few days ago all my FGD212 and FGS223 units stopped working remotely - they work just fine when pressing the buttons at home but not via my iPhone or automations. 

Wondering if anyone else has had any problems with them lately? All other units in my z-wave network work just fine... several brands. Struggling to fill in bug report so I'm testing to ask a question first... ;-) Don't know what has happened - if anything?! 

They all have been working for about four years without any big problems, but I don't know what to do with this now...

**Comment by AlCalzone** [comment]
Well... did you update? Please open a [bug report](https://github.com/zwave-js/node-zwave-js/issues/new?assignees=&labels=&template=bug_report.yml) and fill it in completely. Especially make sure to provide the correct log.

**Reply by petertq** [reply]
I've got the same problem. Can't control my Fibaro switches and dimmers, but if they change status (off to on for example) HA is aware of the change.

It happened after upgrading two or three days ago. I was excited to see that there was a new version available just a few hours ago, but it didn't solve the problem. I'll file a bug report.

**Reply by bckmrk** [reply]
Don't know exactly when the issue first occurred at my place, but the problem description fits mine. It was recently, however.

Struggling to get the right log files out, since I'm not that "in to" this. The log-file-guides look completely different compared to my HA :D

**Reply by AlCalzone** [reply]
https://www.home-assistant.io/integrations/zwave_js#how-to-access-the-z-wave-logs

**Comment by ORCHORDS** [comment]
You're hitting a known Fibaro firmware behavior, not a zwave-js regression — and "worked for four years, then stopped" is the fingerprint.

The driver sends a Multilevel Switch CC with a duration field ([MultilevelSwitchCC.ts:230-249](https://github.com/zwave-js/node-zwave-js/blob/master/packages/cc/src/cc/MultilevelSwitchCC.ts#L230)). When the duration is "default", the firmware of the FGD212/FGS223 decides what to do. After Fibaro's 25.x firmware line rolled out more aggressive auto-calibration, several things converge:

1. The dimming ramp time (parameter 1, default 0 = instant) gets re-clobbered by a forced re-interview after the firmware changes its advertised values. If the interview writes back a non-zero ramp time, every "On" from the hub becomes a slow ramp that *looks* like nothing happened for the first second.

2. Auto-calibration parameter 13 on FGD212 / parameter 13 on FGS223 — when triggered by a power event (brownout, breaker trip, even a heavy load like a hair dryer on the same circuit), it locks out switch commands for 30 s while it re-measures the load. The button still works because the lockout only affects incoming radio commands.

3. S2 security re-key after too many NACKs (which happens if the radio link is marginal, e.g. a new appliance causing RF noise): the node stops accepting encrypted SETs for ~20 s while it renegotiates.

**What to do, in order:**

- Open the device → *Configuration* → check param 1 (dimming ramp) on the FGD212, param 1 + 13 on the FGS223. Set param 1 explicitly to 0 if it has drifted. Set param 13 on the dimmer to 0 to disable auto-calibration if your wiring is stable.
- *Re-interview* the device; the driver will write the discovered config back and you can verify nothing changed unexpectedly.
- If the FGS223 was included with S2, re-include without security (zwave-js supports legacy inclusion when S2 fails repeatedly).
- Check your controller logs for "Dropping message because it could not be deserialized" or "No callback received" around the time the iOS app commands are sent — that confirms an RF/S2 issue, not a device fault.

If you can capture a driver.log (set logLevel "debug" for that node) showing the SET being sent and the FGD212/FGS223's reply, we can tell you definitively whether the SET is reaching the device and which of the three causes it is.
